core/templates/dev/head/pages/questions_list/QuestionsListDirective.ts

```diff
@@ -105,11 +113,20 @@ oppia.directive('questionsList', [
             }
             if (!$scope.questionIsBeingUpdated) {
               EditableQuestionBackendApiService.createQuestion(
-                $scope.skillId, $scope.question.toBackendDict(true)
-              ).then(function() {
-                $scope.fetchQuestionSummaries($scope.entityId, true);
-                $scope.questionIsBeingSaved = false;
-                $scope.currentPage = 0;
+                $scope.skillIds[0], $scope.question.toBackendDict(true)
+              ).then(function(response) {
+                if ($scope.skillIds.length > 0) {
+                  for (var i = 1; i < $scope.skillIds.length; i++) {
```

Comment:
Instead of calling  EditableQuestionBackendApiService.addQuestionSkillLink multiple times, maybe create a function in EditableQuestionBackendApiService (e.g.: addMultipleQuestionSkillLInks) which can add multiple question skill links?
I dont think it's a good idea to keep pinging the backend so many times in succession.

Reply:
Done.